benchmarks/bench_examples.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sys import path
+from os.path import dirname as dir
+
+path.append(dir(path[0]))
+
+import examples.bernstein_vazirani
+import examples.place_on_bristlecone
+import examples.hello_qubit
+import examples.bell_inequality
+import examples.quantum_fourier_transform
+import examples.bcs_mean_field
+import examples.grover
+import examples.phase_estimator
+import examples.quantum_teleportation
+import examples.superdense_coding
+
+
+class ExamplesTest:
```

Comment:
We should have comments here or at the top of the file to explain what this file is doing.  Same with bench_linalg_decompositions.py

Reply:
Hmm, so I could write a trivial comment like "Performance benchmarks for examples/*", but I think the situation is similar to how for every <filename>.py, we write unit tests in <filename>_test.py and don't add explicit comments unless needed otherwise. 

Maybe a benchmarks/README.md, explaining the setup and how to write / run tests, would be more appropriate? (I plan to add this in part-2 of the commit)

Reply:
How about explaining more of the context?  There is no explanation of how to use this benchmark or its connection to "asv" or anything.  Pointing to a package-level README.md is okay.

Reply:
added a package-level README.md explaining the setup and usage of benchmarks.

Reply:
added docstring.